BrewMesModulesParent/common/src/main/java/com/brewmes/common/entities/Connection.java

```diff
@@ -0,0 +1,42 @@
+package com.brewmes.common.entities;
+
+public class Connection {
+    private int id;
+    private String ip;
+    private String name;
+
+    public Connection(int id, String ip, String name) {
```

Comment:
Bør dette være et String id i stedet for int?

Reply:
Tjek igen